SA3/qa_web/forms.py

```diff
@@ -36,3 +36,62 @@ class QuestionsForm(forms.Form):
 class AnswersForm(forms.Form):
     content = forms.Textarea()
 
+class UserProfile(forms.Form):
+    # a form is created to intake the user's personal information
+    prename = forms.CharField(
+        required=True,
+        label='Prename',
+        error_messages={'required': 'Please insert your first name.'},
+
+    )
+
+    surname = forms.CharField(
+        required=True,
+        label='Surname',
+        error_messages={'required': 'Please insert your last name.'},
+
+    )
+
+    email = forms.CharField(
+        required=True,
+        label='E-Mail:',
+        error_messages={'required': 'Please insert your last e-mail.'},
+
+    )
+    age = forms.IntegerField(
+        required=True,
+        label='Age',
+        error_messages={'required': 'Please insert an age.'},
+
+    )
+
+    birthday = forms.DateField(
```

Comment:
This form field is subject to many possible errors that might occur as seen below. We must tell the user to use a correct date format (YYYY-MM-DD) or insert a calendar widget which will allow them to select a correct date. The latter can be an improvement, but the least we can do is add **YYYY-MM-DD** to the label or as a side note.

![image](https://user-images.githubusercontent.com/29107916/36575816-ff9b9cd8-181a-11e8-9fd7-292a80ce7736.png)

Reply:
i left a notice for the user, so he or she is aware of the format.